kibbeh/src/modules/room/chat/RoomChatInput.tsx

```diff
@@ -113,86 +113,61 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({ users }) => {
   };
 
   return (
-    <form onSubmit={handleSubmit} className={`pb-3 px-4 pt-5 flex flex-col`}>
+    <form onSubmit={handleSubmit} className={`pb-3 px-4 pt-2 flex flex-col`}>
       {isEmoji ? (
-        <Picker
-          set="apple"
-          onSelect={(emoji: CustomEmote) => {
-            position =
-              (position === 0
-                ? inputRef!.current!.selectionStart
-                : position + 2) || 0;
-
-            const newMsg = [
-              message.slice(0, position),
-              "native" in emoji
-                ? emoji.native
-                : (message.endsWith(" ") ? "" : " ") +
-                  (emoji.colons || "") +
+        <div className={`mb-1`}>
+          <EmojiPicker
+            emojiSet={customEmojis}
+            onEmojiSelect={(emoji) => {
+              position =
+                (position === 0
+                  ? inputRef!.current!.selectionStart
+                  : position + 2) || 0;
+
+              const newMsg = [
+                message.slice(0, position),
+                (message.endsWith(" ") ? "" : " ") +
+                  (`:${emoji.short_names[0]}:` || "") +
                   " ",
-              message.slice(position),
-            ].join("");
-            setMessage(newMsg);
-          }}
-          style={{
-            position: "relative",
-            width: "100%",
-            minWidth: "278px",
-            right: 0,
-            overflowY: "hidden",
-            outline: "none",
-            alignSelf: "flex-end",
-            margin: "0 0 8px 0",
-          }}
-          sheetSize={32}
-          theme="dark"
-          custom={customEmojis}
-          emojiTooltip={true}
-          showPreview={false}
-          showSkinTones={false}
-          i18n={{
-            search: t("modules.roomChat.search"),
-            categories: {
-              search: t("modules.roomChat.searchResults"),
-              recent: t("modules.roomChat.recent"),
-            },
-          }}
-        />
+                message.slice(position),
+              ].join("");
+              setMessage(newMsg);
+            }}
+          />
+        </div>
       ) : null}
       <div className="flex items-stretch">
-        <div className="flex-1 mr-2 lg:mr-0 items-end">
+        <div className="flex-1 mr-2 lg:mr-0 items-center bg-primary-700 rounded-8 flex-row">
```

Comment:
```suggestion
        <div className="flex-1 mr-2 lg:mr-0 items-center bg-primary-700 rounded-8">
```
flex-row is set by default